app/uk/gov/hmrc/gform/config/ConfigModule.scala

```diff
@@ -101,7 +101,7 @@ class ConfigModule(val context: ApplicationLoader.Context, playBuiltInsModule: P
       } yield s"$url$projectId.js",
       trackingConsentSnippet = new hmrcTrackingConsentSnippet(new TrackingConsentConfig(playConfiguration)),
       emailAuthStaticCodeEmails =
-        getOptionalNonEmptyCIStringList(playConfiguration.getOptional[String]("emailAuth.staticCodeEmails"))
+        getOptionalNonEmptyCIStringList(playConfiguration.getOptional[String]("emailAuth.staticCodeEmails:q"))
```

Comment:
This looks like someone trying to quit vim error?

Reply:
he he...good spot and good guess :D

Reply:
I will be doing another quick fix Jonathan asked for, spacing in the text shown on confirm code page

Reply:
Fix has been added